test/test.ini

```diff
@@ -54,6 +54,11 @@ replication = port=6666 host=127.0.0.1 dbname=replication
 
 client_limit_db= port=6666 host=127.0.0.1 dbname=p0 max_db_client_connections=2
 client_limit_db_auth_passthrough = port=6666 host=127.0.0.1 dbname=p0 auth_user=pswcheck max_db_client_connections=2
+primary_first = port=6666 host=127.0.0.1,127.0.0.2 dbname=p0 user=bouncer target_session_attrs=primary
+primary_second = port=6666 host=127.0.0.2,127.0.0.1 dbname=p0 user=bouncer target_session_attrs=primary
+standby_first = port=6666 host=127.0.0.2,127.0.0.1 dbname=p0 user=bouncer target_session_attrs=standby
+standby_second = port=6666 host=127.0.0.1,127.0.0.2 dbname=p0 user=bouncer target_session_attrs=standby
```

Comment:
Needs some tests for read-only, read-write, and any.

Reply:
Ah yup and those should catch this https://github.com/pgbouncer/pgbouncer/pull/1179/files#r1881005914

Reply:
cabc3ee6c6e60fefb99db029bca0b1ca495656a6